src/api/RequestApi.ts

```diff
@@ -0,0 +1,198 @@
+import { spWebContext } from "../providers/SPWebContext";
+import { ApiError } from "./InternalErrors";
+import { IItemUpdateResult } from "@pnp/sp/items";
+import { emptype } from "../constants/EmpTypes";
+import { worklocation } from "../constants/WorkLocations";
+import { SPPersona } from "../components/PeoplePicker/PeoplePicker";
+
+// create PnP JS response interface for the InForm
+// This extends the IInForm -- currently identical, but may need to vary when pulling in SPData
+type IResponseItem = IInForm;
```

Comment:
At some point we should probably rename the IInForm type to something more descriptive since it's not really describing a form, but some data.

Reply:
Like the thought here....  will probably be something like IInRequest -- looking at making the RequestAPI rebust so will have something like IOutRequest as well.

Reply:
Left this here as a placeholder -- as we actually start putting data into SP, and pulling it from SP, we will have to have this type that converts between the two, right?  Because I can't store a luxon date straight to SP etc.

Reply:
Good call. Will need something to mediate between our exported types used in the app, and some type that will match with what SP actually sends/receives